docs/whatsnew/1.2.rst

```diff
@@ -29,12 +29,14 @@ smaller improvements and bug fixes, which are described in the
 Solar system ephemerides
 ------------------------
 
-Positions of the major solar-system bodies, including the moon, can now be
-calculated.  These integrate with the coordinates subpackage, allowing use of
-apparent (e.g. ``AltAz``) positions or barycentric values.  These can be
-calculated using either using built-in approximations or more precisely
-using JPL ephemeris models (the latter requiring the ``jplephem`` package).
-For more details, see :ref:`astropy-coordinates-solarsystem`.
+:mod:`~astropy.coordinates` can now calculate the positions of the major
+solar system bodies (as well as the moon).  These integrate fully all the
+coordinate frames, allowing easy conversion to apparent (e.g. ``AltAz``)
+positions or barycentric values.  The positions can be calculated using either
+using built-in approximations or more precise values that depend on downloading
+JPL-provided ephemeris models derived from n-body simulations (the latter
+requires the additional dependency of the ``jplephem`` package). For more
```

Comment:
Could you change ``jplephem`` to a link? (I know it was that way before, but since you're updating this paragraph anyway)